__init__.py

```diff
@@ -662,19 +662,23 @@ def register():
     bpy.app.handlers.load_post.append(onLoadFile)
     bpy.app.handlers.undo_post.append(onUndo)
 
-    # Kill ArtNet Thread at exit
-    atexit.register(DMX_ArtNet.disable)
+    # since 2.91.0 unregister is called also on Blender exit
+    if (2, 91, 0) > bpy.app.version:
```

Comment:
```suggestion
    if bpy.app.version <= (2, 91, 0):
```
The project code style prefers the known value to the right on comparisons. What do you think?